frontend/packages/ux-editor/src/components/config/FormComponentConfig.tsx

```diff
@@ -168,6 +205,36 @@ export const FormComponentConfig = ({
             />
           );
         }
+      })}
+
+      {/** Object properties */}
+      {objectRestPropertyKeys.map((propertyKey) => {
+        if (unsupportedPropertyKeys.includes(propertyKey) || !rest[propertyKey]) return null;
+        if (rest[propertyKey].type === 'object' && rest[propertyKey].properties) {
```

Comment:
Wont this be the case for all the properties in the `objectRestPropertyKeys` already?